tdns/delegation_sync.go

```
@@ -216,37 +250,21 @@ func (zd *ZoneData) DelegationSyncSetup(kdb *KeyDB) error {
 			}
 			resp, err := zd.KeyDB.Sig0KeyMgmt(nil, kp)
 			if err != nil {
-				return fmt.Errorf("DelegationSyncSetup(%s) failed to generate keypair: %v", zd.ZoneName, err)
+				return fmt.Errorf("Sig0KeyPreparation(%s) failed to generate keypair: %v", name, err)
 			}
 			zd.Logger.Printf(resp.Msg)
```

golangci-lint (1.61.0)

printf: non-constant format string in call to (*log.Logger).Printf

(govet)